lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -116,7 +126,26 @@ function dosomething_metatag_add_metatag_image($node = NULL) {
         "content" => $image,
       ),
     );
-    drupal_add_html_head($element, 'facebook_share_image');
+    drupal_add_html_head($element, 'og:image');
+
+    $element = array(
+      '#tag' => 'meta',
+      '#attributes' => array(
+        "property" => "og:image:width",
+        "content" =>  740,
+      ),
+    );
+    drupal_add_html_head($element, 'og:image:width');
+
+    $element = array(
+      '#tag' => 'meta',
+      '#attributes' => array(
+        "property" => "og:image:height",
+        "content" =>  480,
+      ),
+    );
+    drupal_add_html_head($element, 'og:image:height');
```

Comment:
@angaither Facebook suggests that we include these dimensions so the image doesn't have to load asynchronously as a part of it's [best practices](https://developers.facebook.com/docs/sharing/best-practices#precaching). 740x480 is the crop we use to build the image URL. I can either hard code here, or try to extract it from the string we pass the function that builds the image URL.